pkg/render/config.go

```diff
@@ -42,12 +43,17 @@ type AWSConfig struct {
 	WithCreds             bool   `json:"withCreds"`
 }
 
+type OpenStackConfig struct {
```

Comment:
this can be dropped. Then and as we move all machiineSets spec the discrimination for the controller should be done `Provider`